plugins/modules/win_dns_record.py

```diff
@@ -19,6 +19,20 @@
     - The name of the record.
     required: yes
     type: str
+  port:
+    description:
+    - The port number of the record.
+    - Optional for C(type=SRV).
+    - Supported only for C(type=SRV).
+    default: ""
+    type: int
+  priority:
+    description:
+    - The priority number for each service in SRV record.
+    - Optional for C(type=SRV).
+    - Supported only for C(type=SRV).
+    default: "0"
```

Comment:
No need to quote an integer

Reply:
when I was writing `default: 0` it was causing the mismatch with the ps1 file. The actual error was "doc-default-does-not-match-spec ". In the doc default, it is taking it as None and in the ps1 file, I assigned it as 0.

Reply:
The default value type should match the type of the option which in this case is an int. So the default in the doc should be `default: 0` and the default value in the arg spec is `default = 0`.

Reply:
Yes, I tried exactly same in the first commit and facing the "doc-default-does-not-match-spec" error. Which is why I quote the above integer.

Reply:
Fixed the changes